### ISSUE TITLE
Known Series Not Appearing (thetvdb)

### ISSUE THREAD

**Issue by Haravikk** [griff/metaz#149]
For some reason I'm not seeing metadata search results from thetvdb for a particular show, even though I know it's listed in their database (I can find it on their website) and I can tag other shows without issue, so it's not a general fault.

The show in question is The Frankenstein Chronicles (new in the UK); I've selected TV Show as the type, entered the show's name and entered a series number, which is usually all that's needed before deselecting then reselecting the file to trigger the search, but no results are appearing.

As I say, I'm still able to tag other shows without issues, it just seems to be this particular show that's causing problems, requiring me to enter all the details manually. I know it could be a problem with thetvdb's API, but I don't know how to debug it so I thought it best to report it here.

**Comment by griff** [comment]
My guess is that it is a bug in their database. I checked it out and when you use Frankenstein Chronicles as the show name (dropping the The) it finds the show just fine

**Comment by ed-kz** [comment]
Does it happen that no TV Episodes, even those previously found and tagged already by metaz, show any tvdb hits? Movies show fine, both newly added and those already tagged. 

It was working recently on mac OS Mojave, with metaz beta 35(?), then stopped just yesterday when I went to use it again. I then looked and downloaded/installed beta 49.  It didn't help.  

Did tvdb change their database perhaps or is it an OS thing? I will see about trying an older mac OS machine if I can find one.

**Comment by griff** [comment]
@ed-kz right now TheTVDB is not responding to requests on the old API that MetaZ uses. And I can't test their new API because their page where you generate API keys doesn't work. It might mean that TheTVDB is having some downtime or it might mean that they have shut down the old API.

**Comment by Haravikk** [comment]
@griff: Seems to be working again with the existing API, perhaps the inability to create codes for the new one was in some way linked to a general problem?

That said I don't seem to be receiving poster images for shows that I'm tagging, just the basic data, but poster fetching has always been a bit hit and miss depending upon the show(s).

**Comment by dagronf** [comment]
Yeah I was able to use MetaZ today with tvdb on The Orville, including posters and all data.

Might just have been that they had downtime on the v1 api.
